lib/index.ts

```diff
@@ -43,6 +41,8 @@ import {
     ConversionShortcutsValidator,
     LocalizationValidator,
 } from './validators';
+import { listFilesRelativeToFolder } from './util/files';
+import { PackageValidator } from './validators/package-validator';
```

Comment:
Is there a reason why this validator is imported separately?

Reply:
No, fixed it :-)